src/module-vsbridge-indexer-core/Console/Command/RebuildEsIndexCommand.php

```diff
@@ -121,7 +121,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (!empty($invalidIndices)) {
             $message = 'Some indices has invalid status: '. implode(', ', $invalidIndices) . '. ';
-            $message .= 'Please change indices status to VALID manually.';
+            $message .= 'Please change indices status to VALID manually or php bin/magento vsbridge:resetindex.';
```

Comment:
change text to `or run php bin/magento vsbridge:resetindex` or `or use php bin/magento vsbridge:resetindex`

Reply:
done

Reply:
done